packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import React, { useCallback } from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { IModalStyles, Modal, Stack, useTheme, Text, IconButton, Icon } from '@fluentui/react';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import {
+  themedPhoneInfoModalStyle,
+  titleClassName,
+  titleContainerClassName,
+  phoneInfoTextStyle,
+  phoneInfoIcon,
+  phoneInfoInctructionLine,
+  phoneInfoStep
+} from './styles/TeamsMeetingConferenceInfo';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { _preventDismissOnEvent } from '@internal/acs-ui-common';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { useLocale } from '../localization';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * Information for conference phone info
+ */
+export interface _ConferencePhoneInfo {
```

Comment:
Is this type something that we need to export externally? does Contoso have the opportunity to interact with this data from the adapter or stateful client?

Reply:
No, i've used this class just for component implementation to display as list. Stateful client has object type from SDK to contains this data.